java/src/jmri/jmrix/powerline/SerialTrafficController.java

```diff
@@ -68,6 +68,22 @@ public void sendX10Sequence(X10Sequence s, SerialListener l) {
      */
     public void sendInsteonSequence(InsteonSequence s, SerialListener l) {
     }
+    
+    /**
+     * Send a sequence of DMX messages to an adapter.
+     * <p>
+     * Makes them into the local messages and then queues in order.
+     * <p>
+     * This is a default, null implementation, which must be overridden in an
+     * adapter-specific subclass.
+     *
+     * @param unitid sequence to send
+     * @param newStep listener for reply
```

Comment:
I don't understand this comment. Is `newStep` a listener? It doesn't seem that to me.

The method `SpecificTrafficController.setDmxIntensity()` has the parameter `intensityValue`.
`@param intensityValue value to put in dmxArray`